css/style.css

```diff
@@ -0,0 +1,1140 @@
+  background: url(../img/tesla_logo_car.png) no-repeat;
+  background-size: contain;
+  margin-top: 134px;
+}
+
+/* .main__img {
+  position: relative;
+  margin-top: -144px;
+  width: 100%;
+} */
+
+.main__title {
+  position: relative;
+  font-weight: 600;
+  margin-top: -76px;
+  font-size: 90px;
+  text-align: center;
+  margin-bottom: -35px;
+  line-height: 110px;
+}
+
+.title-second {
+  font-weight: 500;
+  font-size: 15px;
+  margin-top: 24px;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+}
+
+.main__description {
+  text-align: center;
+  left: 600px;
+  font-size: 15px;
+  margin-top: 27px;
+  margin-bottom: 26px;
+}
+
+.main__container {
+  display: flex;
+  margin-top: 510px;
+  padding: 0 250px;
+  max-width: 100%;
+  align-items: center;
+  text-align: center;
+  justify-content: space-around;
+}
+
+.main__container-first {
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__container-second {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  bottom: 14px;
+  margin-right: 33px;
+}
+
+.main__row {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  justify-content: space-between;
+  margin: 0 62px 0 26px;
+}
+
+.main__img-vector {
+  float: left;
+  padding-left: 10px;
+}
+
+.main__img-stroke {
+  float: left;
+  width: 41.83px;
+  margin-right: -10px;
+}
+
+.main__row-title {
+  display: flex;
+  align-items: baseline;
+  padding-left: 15px;
+  margin-left: -15px;
+  font-style: bold;
+  font-weight: 600;
+  font-size: 35px;
+  color: black;
+  position: relative;
+  right: 3px;
+  margin-top: -10px;
+  font-family: "Montserrat", sans-serif;
+  line-height: 43px;
+}
+
+.main__subtitle-time {
+  display: flex;
+  margin-left: 6px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  color: black;
+  line-height: 24px;
+}
+
+.main__subtitle-dist {
+  display: flex;
+  margin-left: 6px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  color: black;
+  line-height: 24px;
+}
+
+.main__row-desc {
+  display: flex;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+  color: black;
+  position: relative;
+  right: -10px;
+  margin-top: 0;
+  margin-left: 15px;
+}
+
+.main__row-test {
+  display: flex;
+  margin-bottom: 10px;
+  align-items: center;
+  flex-direction: row-reverse;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.row__test {
+  display: flex;
+}
+
+.main__img-steering {
+  float: left;
+  margin-top: 20px;
+  margin-right: 8px;
+}
+
+.main__img-steering-1, .main__img-steering-2 {
+  display: none;
+}
+
+.main__row-text {
+  display: flex;
+  flex-wrap: row nowrap;
+  align-items: center;
+  font-weight: bold;
+  text-align: center;
+  font-size: 13px;
+  color: black;
+  margin-top: 23px;
+}
+
+.button {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  border: 2px solid #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)